core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1135,6 +1136,71 @@ object TestUtils extends Logging {
   }
 
   /**
+   * Verifies that this ACL is the secure one. The
+   * values are based on the constants used in the
+   * ZooKeeper code base.
+   */
+  def isAclSecure(acl: ACL, sensitive: Boolean): Boolean = {
+    info(s"ACL $acl")
```

Comment:
I know this was already like this before, but maybe this and the other case should be `debug`?